e2e-tests/cypress/support/common-methods/clickProjectLink.js

```diff
@@ -2,7 +2,7 @@ module.exports = (projectName) => {
 	cy.get('td:nth-child(1)').each(($e1, index) => {
 		const text = $e1.text();
 		if (text.includes(projectName)) {
-			cy.get('td:nth-child(1)').eq(index).click();
+			cy.get('td:nth-child(1)').eq(index).click(15, 20);
```

Comment:
I think the issue here is that the query selector is getting the `td` not the actual `a` within it. Clearly up until now, that has worked but i wonder if revising the selector is the "proper" fix.

I did a quick test with this and seemed to work:
```
module.exports = (projectName) => {
	cy.get('td:nth-child(1)').contains(projectName).click();
};
```

Reply:
This works great, will use, thanks